docs/usage/customization-guide.md

```diff
@@ -355,6 +355,34 @@ included in the list of accepted features.
 > tag is only evaluated in each re-discovery period, and the expiration of
 > node labels is not tracked.
 
+To exclude specific features from the `local.feature` Feature, you can use the
+`# +no-feature` directive. The same applies to `local.label` Feature where you can
+exclude some features using the `# +no-label` directive.
```

Comment:
We need to also mention that no labels are generated with `+noLabel`

```suggestion
`# +no-feature` directive. The `# +no-label` directive causes the feature to be excluded from the
`local.label` Feature and a node label not to be generated.
```

or smth

Note: now that we have already three directives we could add e.g. a table to list them all. But this can be left as a future enhancement.